frontend/src/helpers/utils.js

```diff
@@ -58,6 +58,10 @@ export const formatDateToMonthDayYear = (date) => {
     return new Date(date).toLocaleString("en-US", { month: "long", day: "numeric", year: "numeric" });
 };
 
+// Is there a better place for this?
```

Comment:
maybe in a `StatusTag.helpers.js` in the same directory as the component unless its used in other places so then `/helpers/tag.helpers.js`

Reply:
It is used elsewhere in the agreement list filters and upcoming draft toggles in other places.
I'll leave here for now, but I am starting to wonder if we need another place for biz logic organized by class (Agreement, BudgetLineItem).